packages/arb-token-bridge-ui/src/components/App/AppContext.tsx

```diff
@@ -6,7 +6,6 @@ type AppContextState = {
     isTransferring: boolean
     isTransactionHistoryPanelVisible: boolean
     isTransactionHistoryShowingSentTx: boolean
-    isPreferencesPanelVisible: boolean
```

Comment:
Preferences (called Settings now) will use queryParams to able to open it after the page reloads when user adds a chain